docker-images/kafka/scripts/zookeeper_run.sh

```diff
@@ -22,7 +22,8 @@ export ZOOKEEPER_DATA_DIR=$ZOOKEEPER_VOLUME$ZOOKEEPER_DATA_BASE_NAME
 mkdir -p $ZOOKEEPER_DATA_DIR
 
 # Create myid file
-echo $ZOOKEEPER_ID > $ZOOKEEPER_DATA_DIR/myid
+echo "$ZOOKEEPER_ID" > $ZOOKEEPER_DATA_DIR/myid
```

Comment:
Why are we changing working stuff?

Reply:
This is my over zealous bash linter again. I will change it back.